app/gui/view/component-browser/component-list-panel/grid/src/entry.rs

```diff
@@ -442,8 +445,8 @@ impl grid_view::Entry for View {
             eval layout_data ((((kind, style, grid_style), entry_sz))
                 data.update_layout(*kind, style, grid_style, *entry_sz)
             );
-            out.contour <+ layout_data.map(|((kind, _, grid_style), entry_sz)| {
-                Data::contour(*kind, grid_style, *entry_sz)
+            out.contour <+ layout_data.map(|((_, _, grid_style), entry_sz)| {
+                Data::contour(grid_style, *entry_sz)
```

Comment:
We ignore half of the parameters. I would replace `layout_data.map` with `all_with(grid_style, input.set_size)`.